src/main/scala/com/cloudera/sparkts/TimeSeriesRDD.scala

```diff
@@ -227,7 +227,15 @@ class TimeSeriesRDD[K](val index: DateTimeIndex, parent: RDD[(K, Vector)])
    * Gets stats like min, max, mean, and standard deviation for each time series.
    */
   def seriesStats(): RDD[StatCounter] = {
-    map(kt => new StatCounter(kt._2.valuesIterator))
+      map(kt => new StatCounter(kt._2.valuesIterator))
```

Comment:
Thinking about it now, I think it actually makes sense to include the keys with the stats in all cases, i.e. to just change the return type of `seriesStats`.